src/o365/spo/commands/feature/feature-enable.ts

```diff
@@ -0,0 +1,154 @@
+import { ContextInfo } from '../../spo';
+import request from '../../../../request';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandValidate,
+} from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  url: string;
+  featureId?: string;
+  scope?: string;
+  force?: boolean;
+}
+
+class SpoFeatureEnableCommand extends SpoCommand {
+  public get name(): string {
+    return commands.FEATURE_ENABLE;
+  }
+
+  public get description(): string {
+    return 'Enable feature for the specified site or web';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.featureId = (!(!args.options.featureId)).toString();
+    telemetryProps.url = (!(!args.options.url)).toString();
+    telemetryProps.scope = (!(!args.options.scope)).toString();
+    telemetryProps.force = args.options.force || false;
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    let scope = args.options.scope;
+    let force = args.options.force;
+    
+    if(!scope) {
+      scope = "web";
+    }
+    if(!force){
+      force = false;
+    }
+
+    if (this.verbose) {
+      cmd.log('Retrieving request digest');
+    }
+    this.getRequestDigest(args.options.url)
```

Comment:
Any chance to have tested it without requestDigest? It seem to have worked for me without it while testing the command. FYI: I am following the Waldek [PR checklist ](https://github.com/pnp/office365-cli/wiki/PR-checklist)  where is stated we better skip the requestDigest if it is not a must.

Reply:
your correct, it works without! Fixed and committed